targettests/execution/angled_gate.cpp

```diff
@@ -6,7 +6,7 @@
  * the terms of the Apache License 2.0 which accompanies this distribution.    *
  ******************************************************************************/
 
-// RUN: nvq++ %cpp_std --target braket --emulate %s -o %t && %t | FileCheck %s
+// RUN: if $braket_avail; then nvq++ %cpp_std --target braket --emulate %s -o %t && %t | FileCheck %s; fi
```

Comment:
Heads up: I added a couple of new braket tests in this PR: https://github.com/NVIDIA/cuda-quantum/pull/2431, so they would need to be updated as well if my PR merges first.